packages/server/lib/modes/run.js

```diff
@@ -1120,27 +1118,31 @@ module.exports = {
 
       obj.shouldUploadVideo = suv
 
-      debug('attempting to close the browser')
+      let videoCaptureFailed = false
+
+      if (endVideoCapture) {
+        await endVideoCapture()
+        .tapCatch(() => videoCaptureFailed = true)
+        .catch(warnVideoRecordingFailed)
```

Comment:
I prefer syntax of `.catch()` to `try {} catch {}`

Reply:
changed the logic to be:
- stop video recording
- close browser
- video compression

previously it was:
- close browser
- stop video recording + video compress